src/forms/CheckboxField.js

```diff
@@ -35,6 +43,10 @@ CheckboxField.propTypes = {
    * Help text used to describe the field's purpose
    */
   help: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+  /**
+   * Sets the initial/default value for this checkbox input field
+   */
+  checkedByDefault: PropTypes.bool,
```

Comment:
Can we keep this consistent with other fields that support same default value and call it defaultValue?